aws/resources/s3_multi_region_access_point.go

```diff
@@ -0,0 +1,83 @@
+package resources
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3control"
+	"github.com/gruntwork-io/cloud-nuke/config"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/cloud-nuke/report"
+	"github.com/gruntwork-io/cloud-nuke/util"
+	"github.com/gruntwork-io/go-commons/errors"
+)
+
+func (ap *S3MultiRegionAccessPoint) getAll(c context.Context, configObj config.Config) ([]*string, error) {
+	accountID, ok := c.Value(util.AccountIdKey).(string)
+	if !ok {
+		logging.Errorf("unable to read the account-id from context")
+		return nil, errors.WithStackTrace(fmt.Errorf("unable to lookup the account id"))
```

Comment:
I was wondering why should be defined dynamic errors and not dedicated struct errors like https://github.com/gruntwork-io/cloud-nuke/blob/master/aws/resources/ec2.go#L621

Reply:
Hey @denis256, I'm not too sure what's the benefit of declaring a dedicated error struct in this scenario. Potential benefit would be using this error code to test things more thoroughly in unit tests -- we are not actually doing that and just checking whether certain scenario produce an error or not. 

Do you have other benefits you have in mind?